lib/app/view/app.dart

```diff
@@ -17,14 +17,14 @@ class App extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    SystemChrome.setPreferredOrientations([
-      DeviceOrientation.portraitUp,
-      DeviceOrientation.portraitDown,
-      if (ExtendedPlatform.isTv) ...[
-        DeviceOrientation.landscapeLeft,
-        DeviceOrientation.landscapeRight,
-      ],
-    ]);
+    // SystemChrome.setPreferredOrientations([
```

Comment:
can you explain why we need to comment this out?

Reply:
fixed